src/ast/passes/codegen_llvm.cpp

```diff
@@ -73,7 +73,7 @@ CodegenLLVM::CodegenLLVM(Node *root, BPFtrace &bpftrace)
                                   Optional<Reloc::Model>()
 #endif
                                       ));
-  target_machine_->setOptLevel(llvm::CodeGenOpt::Aggressive);
+  target_machine_->setOptLevel(llvm::CodeGenOptLevel::Aggressive);
```

Comment:
This will only work on LLVM 18:
```suggestion
#if LLVM_VERSION_MAJOR >= 18
  target_machine_->setOptLevel(llvm::CodeGenOptLevel::Aggressive);
#else
  target_machine_->setOptLevel(llvm::CodeGenOpt::Aggressive);
#endif
```